File: base.database/src/main/java/alvin/base/database/main/views/MainActivity.java
package alvin.base.database.main.views;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import alvin.base.database.R;
import alvin.base.database.dbflow.views.DBFlowActivity;
import alvin.base.database.sqlite.views.SQLiteActivity;
import androidx.appcompat.app.AppCompatActivity;
import butterknife.ButterKnife;
import butterknife.OnClick;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ButterKnife.bind(this);
    }

    @OnClick(R.id.btn_sqlite)
    public void onSQLiteButtonClick(Button button) {
        Intent intent = new Intent(this, SQLiteActivity.class);
        startActivity(intent);
    }

    @OnClick(R.id.btn_dbflow)
    public void onDBFlowButtonClick(Button button) {
        Intent intent = new Intent(this, DBFlowActivity.class);
        startActivity(intent);
    }
}

File: base.kotlin/src/main/java/alvin/kotlin/MainActivity.kt
package alvin.kotlin

import alvin.kotlin.butterknife.ButterKnifeMainActivity
import alvin.kotlin.dbflow.DBFlowMainActivity
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import butterknife.ButterKnife
import butterknife.OnClick

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        ButterKnife.bind(this)
    }

    @OnClick(R.id.btn_butter_knife, R.id.btn_dbflow)
    fun onButtonOkClick(b: Button) {
        var intent: Intent? = null

        when (b.id) {
            R.id.btn_butter_knife -> {
                intent = Intent(this, ButterKnifeMainActivity::class.java)
            }
            R.id.btn_dbflow -> {
                intent = Intent(this, DBFlowMainActivity::class.java)
            }
        }

        if (intent != null) {
            startActivity(intent)
        }
    }
}
